mangaki/mangaki/wrappers/anilist.py

```diff
@@ -97,6 +239,24 @@ def _request(self,
         r.raise_for_status()
         return r.json()
 
+    def list_seasonal_animes(self,
```

Comment:
In this kind of method, you will prefer to enforce keyword-only argument.

This way, it increases readability: `list_seasonal_animes(False, 2017)` vs `list_seasonal_animes(only_airing=False, year=2017)`.

You can add `*` just after `self,` to say: all following arguments are keyword-only.

```
def list_seasonal_animes(self,
                         *,
                         only_airing: …)
```